storage/integration_test.go

```diff
@@ -4404,74 +4389,103 @@ func TestIntegration_Scopes(t *testing.T) {
+			h := testHelper{t}
+			obj := uidSpace.New()
+			o := clientWithCredentials.Bucket(bucket).Object(obj)
+			contents := []byte("test")
+			if err := writeObject(ctx, o, "text/plain", contents); err != nil {
+				t.Fatalf("writing: %v", err)
+			}
+			defer h.mustDeleteObject(o)
+
+			for _, test := range []struct {
+				desc   string
+				opts   SignedURLOptions
+				client *Client
+			}{
+				{
+					desc: "signing with the private key",
+					opts: SignedURLOptions{
+						Method:  "GET",
+						Expires: time.Now().Add(30 * time.Second),
+					},
+					client: clientWithCredentials,
+				},
+			} {
+				t.Run(test.desc, func(t *testing.T) {
+					bkt := test.client.Bucket(bucket)
+					url, err := bkt.SignedURL(obj, &test.opts)
+					if err != nil {
+						t.Fatalf("unable to create signed URL: %v", err)
+					}
 
-	// Create another client to test the sign byte function as well
+					if err := verifySignedURL(url, nil, contents); err != nil {
+						t.Fatalf("problem with the signed URL: %v", err)
+					}
+				})
+			}
+		}, option.WithCredentials(creds))
+}
 
-	scopes := []string{ScopeFullControl, "https://www.googleapis.com/auth/cloud-platform"}
-	ts := testutil.TokenSource(ctx, scopes...)
-	if ts == nil {
-		t.Fatalf("Cannot get token source to create client")
-	}
-	clientWithoutPrivateKey := testConfig(ctx, t, option.WithTokenSource(ts))
-	defer clientWithoutPrivateKey.Close()
+func TestIntegration_SignedURL_Default(t *testing.T) {
+	multiTransportTest(context.Background(), t, func(t *testing.T, ctx context.Context, bucket, _ string, clientWithoutPrivateKey *Client) {
+		h := testHelper{t}
 
-	jwt, err := testutil.JWTConfig()
-	if err != nil {
-		t.Fatalf("unable to find test credentials: %v", err)
-	}
+		jwt, err := testutil.JWTConfig()
+		if err != nil {
+			t.Fatalf("unable to find test credentials: %v", err)
+		}
 
-	// We can use any client to create the object
-	obj := "testBucketSignedURL"
-	contents := []byte("test")
-	if err := writeObject(ctx, clientWithoutPrivateKey.Bucket(bucketName).Object(obj), "text/plain", contents); err != nil {
-		t.Fatalf("writing: %v", err)
-	}
+		obj := uidSpace.New()
+		o := clientWithoutPrivateKey.Bucket(bucket).Object(obj)
+		contents := []byte("test")
+		if err := writeObject(ctx, o, "text/plain", contents); err != nil {
+			t.Fatalf("writing: %v", err)
+		}
+		defer h.mustDeleteObject(o)
 
-	for _, test := range []struct {
-		desc   string
-		opts   SignedURLOptions
-		client *Client
-	}{
-		{
-			desc: "signing with the private key",
-			opts: SignedURLOptions{
-				Method:  "GET",
-				Expires: time.Now().Add(30 * time.Second),
-			},
-			client: clientWithCredentials,
-		},
-		{
-			desc: "signing with the default sign bytes func",
-			opts: SignedURLOptions{
-				Method:         "GET",
-				Expires:        time.Now().Add(30 * time.Second),
-				GoogleAccessID: jwt.Email,
+		for _, test := range []struct {
```

Comment:
nit: this can probably be removed as we can only really have one client per test now, right? So we would never have more than one case here (same for the previous test).